README.md

```diff
@@ -28,6 +30,22 @@ or [environment variables][awsenv]. For more information read [AWS documentation
 [awsenv]: https://docs.aws.amazon.com/cli/latest/userguide/cli-environment.html
 [awsconfig]: https://docs.aws.amazon.com/cli/latest/userguide/cli-config-files.html
 
+#### Stock kubectl vs EKS-vended kubectl
+
+The recommended way to install kubectl on macOS is `brew install kubernetes-cli`. For Kubernetes v1.10, the stock kubectl client works with EKS after installing AWS IAM Authenticator for Kubernetes. Alternatively, you can install Amazon's patched [EKS-vended kubectl binary](https://docs.aws.amazon.com/eks/latest/userguide/configure-kubectl.html) which works with EKS out of the box but lacks automatic upgrades.
```

Comment:
Perhaps something like: 

'Its recommended that you use the stock kubectl (which needs to be v1.10 or higher). It can be installed on macOS using `brew install kubernetes-cli` and for other platforms see the [install instructions](https://kubernetes.io/docs/tasks/tools/install-kubectl/). For this to work you will also need to have the _[AWS IAM Authenticator for Kubernetes](https://github.com/kubernetes-sigs/aws-iam-authenticator)_ installed.

Alternatively, you can install Amazon's patched [EKS-vended kubectl binary](https://docs.aws.amazon.com/eks/latest/userguide/configure-kubectl.html) which works with EKS out of the box **but this lacks automatic upgrades**.'

Reply:
> EKS-vended kubectl binary which works with EKS out of the box

Just like any other version of kubeclt, it's actually an upstream version, they just provide it for convenience in one S3 bucket.